lib/modules/nightMode.js

```diff
@@ -251,14 +259,34 @@ function overrideNightMode() {
 	nightmodeOverrideStorage.set(Date.now());
 }
 
-function isTimeForNightMode() {
+function getGeolocation() {
+	return new Promise(resolve => {
+		navigator.geolocation.getCurrentPosition(position => resolve(position.coords));
```

Comment:
Does `getCurrentPosition` accept a "user denied permission"` callback, so flow can return to the calling function and fail over to `'user'` style?